schema.graphql

```diff
@@ -747,7 +747,7 @@ type CategoryTranslatableContent implements Node {
   id: ID!
   name: String!
   description: JSONString!
-  descriptionJson: String @deprecated(reason: "Will be removed in Saleor 4.0. Use the `description` field instead.")
+  descriptionJson: JSONString @deprecated(reason: "Will be removed in Saleor 4.0. Use the `description` field instead.")
```

Comment:
Are these changes connected to webhooks as well?

Reply:
Not at all, seems the schema was outdated on master :(